tools/pyi/gen_pyi.py

```diff
@@ -243,7 +243,11 @@ def generate_type_hints(fname, decls, is_tensor=False):
                 if a.get('kwarg_only', False) and render_kw_only_separator:
                     python_args.append('*')
                     render_kw_only_separator = False
-                python_args.append(arg_to_type_hint(a))
+                try:
+                    python_args.append(arg_to_type_hint(a))
+                except Exception as e:
+                    print("Error while processing function %s" % fname)
+                    raise e
```

Comment:
Downside of doing this is you lose internal backtrace. Instead, you should `raise` without an argument, which will prefer the backtrace.

```
pmacbook-pro-116:~ ezyang$ python
Python 2.7.16 (default, Mar  4 2019, 09:01:38) 
[GCC 4.2.1 Compatible Apple LLVM 10.0.0 (clang-1000.11.45.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> try:
...   raise RuntimeError("arf")
... except Exception as e:
...   raise e
... 
Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
RuntimeError: arf
>>> try:
...   raise RuntimeError("arf")
... except Exception:
...   raise
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
RuntimeError: arf
```